pkgs/by-name/fe/feluda/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "feluda";
+  version = "2.7.7";
+
+  src = fetchFromGitHub {
+    owner = "anistark";
+    repo = pname;
+    rev = "98fd4a640cec620dee85c595ec1376949bb025e5";
```

please use the version tag. `https://github.com/anistark/feluda/releases/tag/1.3.0`. the version should be 1.3.0 as well. Also, please do not use pname recursively. Also, please use hash with the hash prefix instead of sha256. Example:
```suggestion
    repo = "feluda";
    tag = version;
    hash = "sha256-...";
```